code/modules/projectiles/pins.dm

```diff
@@ -305,13 +305,13 @@ var/list/wireless_firing_pins = list() //A list of all initialized wireless firi
 			var/obj/item/gun/energy/EG = gun
 			if(EG.required_firemode_auth[EG.sel_mode] == WIRELESS_PIN_STUN)
 				return TRUE
-			else if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level == SEC_LEVEL_RED)
+			else if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level >= SEC_LEVEL_BLUE)
 				return TRUE
 			else
 				fail_message = SPAN_WARNING("Unable to fire: insufficient security level.")
 				return FALSE
 		else
-			if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level == SEC_LEVEL_RED)
+			if (GLOB.security_level == SEC_LEVEL_YELLOW || GLOB.security_level == SEC_LEVEL_RED || GLOB.security_level == SEC_LEVEL_BLUE)
```

Comment:
This has still the same formatting as before

Reply:
That's strange. I comitted the suggested changes on my phone .... maybe the phone interface is wack. I'll just apply this change manually and push it with Gitkraken.